test.kt

```
@@ -0,0 +1,88 @@
+    val s = StringBuilder("Test")
+}
+
+fun genericExample(items: List<String>>): MutableList<String> {
+    val result = MutableList<String>()
+    return result
+}
+
+var uninitialized: Double
+
+fun badWhen(x: Int): String {
+    when x {
```

detekt (1.23.8)

[warning] This when block is empty.

(detekt.empty-blocks.EmptyWhenBlock)